src/dashboard/calc_pnl.py

```diff
@@ -134,12 +199,12 @@ def calculate_spot_price_for_position(
         The position duration
     maturity_timestamp : pd.Series
         The maturity timestamp
-    block_timestamp : pd.Series
+    block_timestamp : Decimal
         The block timestamp
     """
     # pylint: disable=too-many-arguments
     full_term_spot_price = calculate_spot_price(share_reserves, bond_reserves, initial_share_price, time_stretch)
-    time_left_seconds = maturity_timestamp - block_timestamp
+    time_left_seconds = maturity_timestamp.apply(lambda x: x - block_timestamp)
```

Comment:
I'm pretty sure this can just be `maturity_timestamp - block_timestamp`, as this is a vector - scalar, which should just broadcast

Reply:
ya it was, I just had a pyright error... maybe `#type: ignore` is better here?